src/schema-generation/mutation-type-generator.ts

```diff
@@ -421,7 +421,9 @@ export class MutationTypeGenerator {
             resolve: () => new EntitiesQueryNode(rootEntityType),
         };
 
-        const fieldWithListArgs = this.listAugmentation.augment(fieldBase, rootEntityType);
+        const fieldWithListArgs = this.listAugmentation.augment(fieldBase, rootEntityType, {
+            orderByAugmentationOptions: { firstLimitCheckType: 'ResultValidator' },
+        });
```

Comment:
We also need to apply the logic to `generateDeleteAllField` (forgot about that when we talked about it). It returns all deleted objects, so we also need to limit it.